index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>my-portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="navbar">
+            <a href="#">
+                <img class="my-logo" src="images/My Logo.png" alt="logo">
+            </a>
+            <a href="#">
+                <img class="menu" src="images/union.png" alt="union">
+            </a>
+        </div>
+    </header>    
+    <div class="main-section">    
+        <h1 class="main-text">I’m Godanah
+            Glad to see you!</h1>
+        <p class="content">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <br>
+        <p>LET'S CONNECT</p>
+        <br>
+        <img src="images/social-media-icons.png" alt="social-media">
+    </div>     
```

Comment:
- [ ] The container of this section should be a `section` tag. You rather used a div tag which is not very descriptive of the element. I would advise the use of semantic tags

Reply:
Thank you, Jane. Let me make the changes